packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -318,6 +314,12 @@ export abstract class FluidDataStoreContext
 		this.mc = createChildMonitoringContext({
 			logger: this.logger,
 			namespace: "FluidDataStoreContext",
+			properties: {
+				all: tagCodeArtifacts({
+					fluidDataStoreId: this.id,
+					fullPackageName: this.pkg?.join("/"),
```

Comment:
I was looking at some telemetry and then code and realized that this may not be correct.
This will always be undefined for data stores that are loaded from a summary. `pkg` will be undefined during construction and is initialized when the data store is realized. Maybe if we can make this a getter or update it when the data store is initialized, it would work.

Reply:
Created AB#5432 to track this.